config/crds/clusterlink.net_imports.yaml

```diff
@@ -67,6 +67,79 @@ spec:
             - port
             - sources
             type: object
+          status:
+            description: Status represents the import status.
+            properties:
+              conditions:
+                description: Conditions of the import.
+                items:
+                  description: "Condition contains details for one aspect of the current
```

Comment:
Is this text auto-generated or manually created?
It seems to contain generic "boilerplate" which isn't too helpful to users (e.g., `FooStatus` and the known conditions).

Reply:
This whole directory is auto-generated.